src/guides/v2.3/graphql/caching.md

```diff
@@ -109,6 +109,25 @@ To enable GraphQL caching on Fastly:
 
 [Set up Fastly]({{ site.baseurl }}/cloud/cdn/configure-fastly.html) describes how to perform both of these tasks.
 
+By default, the Faslty module for Magento provides the follwing VCL configuration for GraphQL caching:
+
+```text
+if (req.request == "GET" && req.url.path ~ "/graphql" && req.url.qs ~ "query=") {
+....
+```
+
+Fastly will only cache GET requests that contain a query parameter in the request url.
+
+### Example
+
+```text
+http://example.com/graphql?query={ products(filter: {sku: {eq: "Test"}}) { items { name } } }&variables={}
+....
+```
+
+{:.bs-callout-info}
+If you call GraphQL queries in the query body rather than the url (e.g. as --data-raw '{"query" .... }'), the request is not cached.
```

Comment:
```suggestion
If you call GraphQL queries in the query body rather than the URL (for example, as `--data-raw '{"query" .... }'`), the request is not cached.
```